crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -208,9 +229,10 @@ impl AccountInterface {
         let note_creation_source = self.build_create_notes_section(output_notes)?;
 
         let script = format!(
-            "begin\n{}\n{}\nend",
+            "begin\n{}\n{}\n{}\nend",
             self.build_set_tx_expiration_section(expiration_delta),
             note_creation_source,
+            self.build_tx_authentication_section()
```

Comment:
Oh I think this and a few other items are results of a merge gone wrong perhaps? These look like items from before #1480